packages/make-styles/src/runtime/resolveStyleRules.test.ts

```diff
@@ -367,6 +367,46 @@ describe('resolveStyleRules', () => {
     });
   });
 
+  it('handles font-face', () => {
+    expect(
+      resolveStyleRules({
+        '@font-face': {
+          fontFamily: 'Open Sans',
+          src: `url("/fonts/OpenSans-Regular-webfont.woff2") format("woff2"),
+               url("/fonts/OpenSans-Regular-webfont.woff") format("woff")`,
+        },
+      }),
+    ).toMatchInlineSnapshot(`
+      @font-face {
+        font-family: Open Sans;
+        src: url("/fonts/OpenSans-Regular-webfont.woff2") format("woff2"),
+          url("/fonts/OpenSans-Regular-webfont.woff") format("woff");
+      }
+    `);
+  });
+
+  it('handles static css', () => {
+    expect(
+      resolveStyleRules({
+        body: {
+          background: 'blue',
+        },
+        '.foo': {
+          background: 'yellow',
+          marginLeft: '5px',
+        },
```

Comment:
IMO, we another API for this.. For example, Emotion handles it differently:

```jsx
<div
  css={{
    ".foo": {
      color: "red"
    },
    div: {
      color: "green"
    }
  }}
/>
```

![Emotion result](https://user-images.githubusercontent.com/14183168/107353922-e2275300-6acd-11eb-99b3-59d92fd6d219.png)

---

What about ":global"?

```jsx
const styles = {
  ":global": {
    body: {
      background: "blue"
    }
  }
};
```

We can also say that `:global` can be used only as root key, while `:global()` does not have such limitations.

Reply:
`:global(body)` looks like a complete magic to me :-(
`:global` in object root seems to be reasonable.

But first of all, what is the motivation to use `makeStyles` API to insert the global styles? How would the API look like if we would have a separate function for the global styles? What would be the drawbacks of that? How would be the keyframes handled?

Reply:
great point @layershifter, I missed this scenario.

@miroslavstastny -
>But first of all, what is the motivation to use makeStyles API to insert the global styles? How would the API look like if we would have a separate function for the global styles?

If we can expand functionality while keeping it clean with 1 API, that would be ideal. But i do see things are getting messier here, let me explore having a separate api (e.g. `makeStaticStyles`).

>How would be the keyframes handled?

There will be essentially 2 ways for defining keyframes.
Option 1 - preferred if the keyframes is used in 1-off place (impl: #16757)
```jsx=
makeStyles([
  [
    null, {
      animationName: {
         // keyframes value here
      },
      ...
    }
  ]
])
```

Option 2 - preferred if the keyframes is shared in many places
```jsx=
makeStaticStyles({
  '@keyframes fade-in': {...}
})

makeStyles([
  [null, {
    animationName: 'fade-in',
    ....
  }]
])
```

Reply:
> Option 2 - preferred if the keyframes is shared in many places

This might create an unclear dependency 🤔 For example, I will use `animationName`, but component that defines it have not been loaded yet, in this case there will be no definition for this animation...

Reply:
I updated the PR description & code to have a separate interface for `makeStaticStyles`. Please give a look :)

>This might create an unclear dependency 🤔 For example, I will use animationName, but component that defines it have not been loaded yet, in this case there will be no definition for this animation...

I think in case of this, user will need to call both hooks in component code, which I think it's an acceptable expectation?

Reply:
> I think in case of this, user will need to call both hooks in component code, which I think it's an acceptable expectation?

Yep, my concern is only that there are no clear connection between these hooks.